glide.yaml

```diff
@@ -30,7 +30,11 @@ import:
   - ssh/terminal
 - package: github.com/projectcalico/go-yaml-wrapper
 # v1.0.5 seems to have a bad interaction with our logging hooks.
+# need to use aliasing because of this https://github.com/sirupsen/logrus/issues/553#issuecomment-306591437
+# and case difference between sirupsen and Sirupsen
```

Comment:
Do we need the aliasing if both are lowercase? Seemed to be getting along fine without this before.